wandb/sdk/artifacts/artifact.py

```diff
@@ -327,6 +302,10 @@ def _from_attrs(
         artifact.metadata = cls._normalize_metadata(
             json.loads(attrs["metadata"] or "{}")
         )
+        artifact._set_gql_to_ttl_duration_seconds(attrs.get("ttlDurationSeconds"))
```

Comment:
Nit: it's much easier to reason about functions that don't change the state of objects. Let's
- make this function return the value instead of setting the attribute, and set the attribute here
- make this function static to make it clear it doesn't change the state of the artifact
- rename it to `_ttl_duration_seconds_from_gql`, since it's a counterpart to `_ttl_duration_seconds_to_gql`
```suggestion
        self._ttl_duration_seconds = Artifact._ttl_duration_seconds_from_gql(attrs.get("ttlDurationSeconds"))
```